docs/dogstatsd/internals.md

```diff
@@ -88,10 +88,16 @@ The TimeSamplerWorker runs in an infinite loop. It is responsible for the follow
 
 The following calculations determine the number of TimeSamplerWorker and TimeSampler instances:
 
-    * If `dogstatsd_pipeline_autoadjust` is `true`, there are `(number of core/2) - 1` instances of TimeSampler.
+    * If `dogstatsd_pipeline_autoadjust` is `true` then the workers count will be automatically adjusted.
     * If `dogstatsd_pipeline_count` has a value, the number of TimeSampler pipelines equals that value.
     * If neither condition above is true, one TimeSampler pipeline runs.
 
+`dogstatsd_pipeline_autoadjust_strategy` can be set to the following values:
+
+    * `max_throughput`: The number of TimeSampler pipelines is adjusted to maximize throughput. There are `(number of core/2) - 1` instances of TimeSampler.
+    * `per_origin`: The number of TimeSampler pipelines is adjusted to minimize latency. The number of dsdWorker instances is equal to half the number of cores.
```

Comment:
> The number of TimeSampler pipelines is adjusted to minimize latency.

Does this improve latency or compression ratio (which is mentioned in the changelog)?